### ISSUE TITLE
can't edit minutes / previous meeting info

### ISSUE THREAD

**Issue by johnabuck** [agilepro/cognoscenti#268]
I cloned the first meeting into the third session meeting because I needed to get a guest instructor started. Now I'm unable to edit the third session to put in minutes from session 2 rather than session 1.

![image](https://user-images.githubusercontent.com/10082365/47611133-838d5680-da35-11e8-98c1-69ac23957c56.png)

**Comment by agilepro** [comment]
Right, there is no way to change that setting.  Requires some work to allow the value to be edited.
